### PR TITLE
Add tags, pagination, and CSS modules

### LINKED ISSUE #33: Add tag pages

### PR DESCRIPTION
**Pull request by craigmdennis**
Closes #33 #35

There's a lot going on 😅 (I'm not great at atomic commits).
Feedback welcome on implementation.

- Added pagination
- Added tags (and pagination)
- Moved to CSS modules
- Added more navigation items

### CODE REVIEW

File: _posts/2020-03-31-use-infinite-scroll/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ permalink: /posts/introducting-use-infinite-scroll-react-hook
 author: Lukáš Mladý
 thumbnail: ''
 metaDescription: ''
-tags: [engineering, react, scroll]
+tags: [engineering, react, scroll, open source]
```
Comment (lukasmlady) [inline _posts/2020-03-31-use-infinite-scroll/index.md:8]: also react-custom-scroller and smart-tooltip-delay and ciso8601 and MongoMallard posts are `open source`

**Review by lukasmlady — APPROVED** [review]
LGTM, awesome that you've put this together!

I've left a suggestion about tagging more blog posts with "open source".

Also, clicking on some tags leads to weirdly capitalized tag names, see below

---

(should be AbortController)
Snímek obrazovky 2020-10-05 v 14 07 02

---

(Should be MongoDB)
Snímek obrazovky 2020-10-05 v 14 06 52

---

(Should be UX)
Snímek obrazovky 2020-10-05 v 14 06 42

File: _posts/2020-03-31-use-infinite-scroll/index.md (hunk 1 above)
Comment (craigmdennis) [reply _posts/2020-03-31-use-infinite-scroll/index.md]: Added.

**Comment by craigmdennis** [comment]
Just noticed an issue with the page title on the homepage.

**Comment by lukasmlady** [comment]
LGTM. in the last commit, you dropped the `meta` tag. was that on purpose?

**Comment by craigmdennis** [comment]
> LGTM. in the last commit, you dropped the `meta` tag. was that on purpose?

Yes, I thought I'd included a default `<Helmet>` config in `layout` but I haven't. Will add the SEO component in another PR. I'll put this back.

**Pull request merged**